Aliases/gazebo10

```diff
@@ -0,0 +1 @@
+../gazebo8.rb
```

Comment:
I think this should be 9, which needs to be changed to have different dependencies than 8

Reply:
pull request #349 is need first.

Reply:
I just merged #349

Reply:
Update to point to gazebo9